TUTORIAL.md

```diff
@@ -232,7 +232,7 @@ Some MVU tips for beginners
 
 ### Normalize your model; use IDs instead of duplicating entities
 
-It is generally recommended that you aggressively normalize your model. This is because everything is (normally) immutable, so if a single entity occurs multiple places in your model and that entity should be updated, it must be updated every place it occurs. This opens up for state synchronization bugs.
+It is generally recommended that you aggressively normalize your model. This is because everything is (normally) immutable, so if a single entity occurs multiple places in your model and that entity should be updated, it must be updated every place it occurs. This opens up opportunities for state synchronization bugs.
```

Comment:
I think "opportunities" is a somewhat weird word to use in this negative context. I suggest keeping it as it is, or otherwise rephrasing (e.g. "This makes it possible to get state synchronization bugs.")

Reply:
Perhaps "This leads to an increased risk of introducing state synchronization bugs."

Reply:
Excellent, let's go for that!